projects/components/src/filtering/filter-bar/filter-bar.component.ts

```diff
@@ -168,7 +171,7 @@ export class FilterBarComponent implements OnChanges, OnInit, OnDestroy {
   }
 
   private resetFocus(): void {
-    this.filterInput?.nativeElement.focus();
+    this.filterInput?.focus();
```

Comment:
This was the main change. We don't want to focus on the filter chip, we want to focus on the input inside it. Exposed focus method down the component stack.